tests/cross_functional/kcs/test_monitor_recovery.py

```
@@ -157,12 +160,45 @@ def test_monitor_recovery(
         logger.info("Updating initial delay on all monitors")
         update_mon_initial_delay()
 
+        logger.info("Copy tar to the MONs")
+        mon_recovery.copy_tar_to_pods(pod_type="mon")
+
+        logger.info("Copy the previously retrieved monstore to the mon-a pod")
+        mon_pods = get_mon_pods(namespace=config.ENV_DATA["cluster_namespace"])
+        mon_a = mon_pods[0]
+        logger.info(f"Copying mon-store into monitor: {mon_a.name}")
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)